playwright-e2e/tests/admin-add-case-flag.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { test } from '../fixtures/create-fixture';
+import { Apihelp } from '../utils/api-helper';
+import caseData from '../caseData/mandatoryWithMultipleChildren.json';
+import {newSwanseaLocalAuthorityUserOne, CTSCUser, judgeUserWithAdminRole} from '../settings/user-credentials';
+import { Page, expect} from "@playwright/test";
+import {AddAndRemoveAdminCaseFlag} from '../pages/add-and-remove-admin-case-flag';
+import {SignInPage} from '../pages/sign-in';
+
+test.describe('Add a case flag', () => {
+    let apiDataSetup = new Apihelp();
+    const dateTime = new Date().toISOString();
+    let caseNumber: string;
+    let caseName: string;
+
+    test.beforeEach(async () => {
+        caseNumber = await apiDataSetup.createCase('e2e case', newSwanseaLocalAuthorityUserOne);
+    });
+
+    test('Add and remove a case flag as admin user',
+        async ({page, signInPage, addAdminCaseFlag}) => {
+            caseName = 'Add and remove a case flag' + dateTime.slice(0, 10);
+            await apiDataSetup.updateCase(caseName, caseNumber, caseData);
+            await signInPage.visit();
+            await signInPage.login(CTSCUser.email, CTSCUser.password);
+            await runTest(signInPage, addAdminCaseFlag, page);
+        });
+
+    test('Add and remove a case flag as judicial admin user',
+        async ({page, signInPage, addAdminCaseFlag}) => {
+            caseName = 'Add and remove a case flag' + dateTime.slice(0, 10);
+            await apiDataSetup.updateCase(caseName, caseNumber, caseData);
+            await signInPage.visit();
+            await signInPage.login(judgeUserWithAdminRole.email, judgeUserWithAdminRole.password);
```

Comment:
can use normal judgeWales here as mentioned in the other comment!

Reply:
done